pages/version-policy/index.md

```diff
@@ -40,14 +40,11 @@ Thus, upon the release of 2.0.0 where 1.10.4 and 1.9.7 are available:
 
 ## DC/OS Platform Version Compatibility Matrix
 
-The following matrix shows the platform components and operating environments on which DC/OS has been tested; DC/OS will run on these components. Customers running DC/OS on non-supported platform components should upgrade to a supported component. For clarity, Mesosphere only provides support services to paying customers under a written agreement; the term “supported” in Mesosphere documentation refers to whether the indicated software component has been tested for compatibility.
+Customers running DC/OS on non-supported platform components should upgrade to a supported component. For clarity, Mesosphere only provides support services to paying customers under a written agreement. The term “supported” in Mesosphere documentation refers to whether the indicated software component has been tested for compatibility.
 
-|Display Icon | Service       |
-|------------ |-------------- |
-| ⚫          | Supported     |
-|             | Not Supported |
-| B           | Beta          |
+The following tables represent the support matrix for platform components and operating environments on which DC/OS has been tested. DC/OS will run on the following platform components. 
```

Comment:
Could you have a link to each of the tables? It would be easier for people to navigate by clicking on the link to the table instead of scrolling down to find the OS they are looking for.

Reply:
Done.